hw/ip/usbdev/data/usbdev_testplan.hjson

```diff
@@ -654,11 +654,11 @@
     {
       name: stall_trans
       desc: '''
-            Verify that by enabling in_stall/out_stall shall cause STALL response to
-            attempted IN/OUT transactions.
+            Verify that by enabling out_stall shall cause STALL response to
```

Comment:
I know this was there already, but would you mind dropping the "by" from this sentence? (Minor grammar nit)